### PR TITLE
Call l3kernel and l3backend components "modules"

### PR DESCRIPTION
**Pull request by muzimuzhi**

### CODE REVIEW

**Review by josephwright — APPROVED** [review]

File: l3kernel/l3fp.dtx [hunk 1]
```diff
@@ -1567,7 +1567,7 @@
 %
 % \section{Disclaimer and roadmap}
 %
-% The package may break down if the escape character is among
+% This module may break down if the escape character is among
```
Comment (car222222) [inline l3kernel/l3fp.dtx:1570]: "break down" sounds a bit strange: just "break" is better!

File: l3kernel/l3basics.dtx [hunk 2]
```diff
@@ -49,8 +49,8 @@
 %
 % \begin{documentation}
 %
-% As the name suggest this package holds some basic definitions which
-% are needed by most or all other packages in this set.
+% As the name suggest this module holds some basic definitions which
```
Comment (car222222) [inline l3kernel/l3basics.dtx:52]: Better: As the name suggests, . . .

File: l3kernel/l3basics.dtx [hunk 3]
```diff
@@ -49,8 +49,8 @@
...
+% are needed by most or all other modules in this set.
 %
 % Here we describe those functions that are used all over the place. With
 % that we mean functions dealing with the construction and testing of
```
Comment (car222222) [inline l3kernel/l3basics.dtx:56]: Better:  By that,

**Review by car222222 — APPROVED** [review]
Phew!  Many thanks for all this! 
I did spot a few improvements in the surrounding texts.

**Comment by josephwright** [comment]
I'll merge then make those related changes

**Comment by car222222** [comment]
OK, thanks!

**Comment by muzimuzhi** [comment]
@car222222 Joseph has committed your suggestions in a0babfdb and 6a33f085.

**Pull request merged**